Standard Gaming Platform/WinFont.h

```diff
@@ -5,6 +5,9 @@
 void InitWinFonts( );
 void ShutdownWinFonts( );
 
+void InitTooltipFonts();
+void ShutdownTooltipFonts();

```

Comment:
All the code I'm touching is written in plain C, while RAII is a pattern that is predicated on using classes.
I was trying to keep consistency with the existing code base.

I can see the appeal of encapsulating the creation part in a constructor and the cleanup in the destructor. The `InitTooltipFonts` would still be required. The `ShutdownTooltipFonts` would become redundant since the destructor would be called at program shutdown, at least according to my understanding.

This could be a stand alone PR. What would you recommend?

Reply:
and that's fine, I'm just suggesting since there's also the case of the freestanding `fTooltipScaleFactor > 1` in `SoldierTooltips.cpp` which kind of seems to want to be in a static method somewhere, it's a clue that these things want to be together.

`InitTooltipFonts` would go in the constructor, `ShutdownTooltipFonts` is the destructor, and it'd be called when its object goes out of scope, which I guess you want to be around line 266 of `ShutdownFonts()`

but sure, it could be another PR.